hr_system/hr_employee_mgt_services.bal

```diff
@@ -0,0 +1,67 @@
+import ballerina/http;
+import ballerina/log;
+
+type InputData {
+    Employee employee;
+    int recordId;
+    string createdDate;
+    string lastUpdated;
+};
+
+type Employee {
+    string name;
+    int age;
+    string city;
+    string employeeId;
+    string departmentId;
+};
+
+endpoint http:Listener EmployeeMgtServiceListener {
+    port:9023
+};
+
+@http:ServiceConfig {
+    basePath: "/hr/employee-mgt"
+}
+service<http:Service> hrEmployeeManagementService bind EmployeeMgtServiceListener {
+
+    @http:ResourceConfig {
+        methods: ["POST"],
+        path: "/employee"
+    }
+    addEmployee(endpoint caller, http:Request addEmployeeReq) {
+        InputData inputData;
+        Employee employee;
```

Comment:
Any reasons to declare these two objects here and assigning values in few lines after?

Reply:
Fixed